tests/unit/test_misc.py

```diff
@@ -58,6 +59,15 @@ def test_port_mappings(self):
         map.add([234, 237], [345, 348])
         self.assertEqual(map.to_str(), '-p 123-124:123-124 -p 234-237:345-348')
 
+    def test_get_service_status(self):
+        env = infra.get_services_status()
+        self.assertNotEqual(env, None)
+        self.assertGreater(len(env), 0)
+
+    def test_update_config_variable(self):
+        env = infra.update_config_variable('foo', 'bar')
+        print(env)
```

Comment:
Let's please remove the `print(..)` statement.

I think we should also add an assertion here.

Add this to the import section at the top:
```
from localstack import config
```

... then this in the test:
```
self.assertEquals(config.foo, 'bar')
```